packages/components/pie-input/src/input.scss

```diff
@@ -4,17 +4,22 @@
     --input-padding-block: var(--dt-spacing-c);
     --input-padding-inline: var(--dt-spacing-d);
     --input-gap: var(--dt-spacing-d);
-    --input-text-clr: var(--dt-color-content-subdued);
+    --input-text-clr: var(--dt-color-content-default);
```

Comment:
I know this is probably just me (and my Friday brain) but I legit spent a whole minute trying to work out what `clr` stood for 😆  I'd probably change these to `color`, as we don't abbreviate in other components (or in the design tokens) so will keep it consistent

Reply:
Oops! Sorry, a misguided assumption from me there 🤦 I will rename them